tests/cli/test_setup_gate.py

```
@@ -109,9 +103,33 @@ def test_other_entry_commands_blocked_pre_setup(
-    # output if the gate didn't fire first.
     result = runner.invoke(app, [cmd])  # no --help so the gate runs
     # Some commands may also error on missing required args; the
     # panel must appear regardless of the secondary failure.
     assert "First-time setup required" in result.output
+
+
+@pytest.mark.unit
+@pytest.mark.ci
+@pytest.mark.uses_setup_gate
+@pytest.mark.parametrize("cmd", ["organize", "search", "analyze", "preview"])
+def test_help_bypasses_gate_pre_setup(cmd: str, fresh_config: Path) -> None:
```

Ruff (0.15.12)

[warning] Unused function argument: `fresh_config`

(ARG001)